posts/How-to-add-blog-post/index.qmd

```diff
@@ -0,0 +1,214 @@
+---
+title: "How to add a blog post using Quarto"
+author: 
+    name: "Richie Moluno, Goodnews Sandygudie"
+date: "2022-12-08"
+categories: 
+    - "blog"
+    - "quarto"
+    - "Outreachy"
+description: "This is a tutorial on how to add a blog post to the OSCA blog using Quarto."
+licence: "CC BY"
+type: "post"
+toc-title: "Table of Contents"
+toc-location: right
+toc-depth: 3
+toc: true
+image: "./quarto.png"
+citation: 
+    url: https://oscsa.netlify.app/posts/how-to-add-a-blog-post/
+site-url: https://oscsa.netlify.app/posts/how-to-add-a-blog-post/
+---
+
+#
+
+![](./quarto.png){fig-align="center" width="500" fig-caption="Quarto logo"}
+
+#
+
+## What is Quarto?
+Quarto is a new open source project that aims to make it easier for people to create and publish blog posts. It is a static site generator that uses the Markdown syntax to create blog posts. Quarto is written in the R programming language and is available for download on GitHub.
+
+## Get started
+### Prerequisites
+To get started with contributing with Quarto, you will need;
+
+- A GitHub account.
+- A basic understanding of Markdown is recommended but not necessary.
+
+### Install Quarto
```

Comment:
I don't think they need to install Quarto - so maybe this paragraph can be moved to the side using `{.aside}`

Reply:
Oh, alright thanks

Reply:
done